src/compiler/csharp_generator_helpers.h

```diff
@@ -25,9 +25,10 @@
 namespace grpc_csharp_generator {
 
 inline bool ServicesFilename(const grpc::protobuf::FileDescriptor* file,
+                             const std::string file_suffix,
                              std::string* file_name_or_error) {
```

Comment:
nit: it's pre-existing and so outside of the scope of this PR, but `std::string* file_name_or_error` should be changed to `std::string& file_name_or_error` per the [style guide's](https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs) suggestion to make non-nullable output params references.

Reply:
Done.